tests/utils/admincli/vmgroupmgmt.go

```diff
@@ -93,13 +121,37 @@ func ConfigInit(ip string) (string, error) {
 	}
 
 	log.Printf("Initializing the SingleNode Config DB on esx [%s] \n", ip)
-	return ssh.InvokeCommand(ip, admincli.InitLocalConfigDb)
+	out, err := ssh.InvokeCommand(ip, admincli.InitLocalConfigDb)
+
+	if err != nil {
+		return out, err
+	}
+
+	// verify the DB init
+	if GetDBmode(ip) == admincli.DBSingleNode {
+		return "DB init successful on esx " + ip, nil
```

Comment:
Shouldn't we return original command output in this case?

```golang
return out, nil
```
Basically the point is: we don't want to change the original command output if the extra verification passes.